src/project-infos.js

```diff
@@ -124,7 +147,7 @@ const getProjectInfos = async () => {
   const description = get(packageJson, 'description', undefined)
   const engines = get(packageJson, 'engines', undefined)
   const author = getAuthorName(packageJson)
-  const version = get(packageJson, 'version', undefined)
```

Comment:
This should not be changed, you can add a new information instead on modifying existing one 😉